### PR TITLE
interface description output

### PR DESCRIPTION
**Pull request by mmacata**
This PR suggests to let the GRASS module interface description be printed via process chain parameter

### CODE REVIEW

**Comment by mmacata** [comment]
This PR is WIP. To be discussed, if we really need this

**Comment by mmacata** [comment]
Not WIP anymore, we need it. It is tested and awaiting review once again ;) @AnikaBettge

File: src/actinia_core/resources/common/process_chain.py [hunk 1]
```diff
@@ -989,6 +991,9 @@ def _create_module_process(self, module_descr):
         if "verbose" in module_descr and module_descr["verbose"] is True:
             params.append("--v")
 
+        if "interface-description" in module_descr and module_descr["interface-description"] == "True":
```
Comment (marcjansen) [inline src/actinia_core/resources/common/process_chain.py]: Just a question, why is it `"True"` here (a string), but for the `verbose` parameter above it is `True` (a boolean)?

File: src/actinia_core/resources/common/process_chain.py [hunk 2]
```diff
@@ -1284,6 +1290,9 @@ def _create_module_process_legacy(self, id, module_descr):
         if "verbose" in module_descr and module_descr["verbose"] is True:
             parameters.append("--v")
 
+        if "interface-description" in module_descr and module_descr["interface-description"] == "True":
```
Comment (marcjansen) [inline src/actinia_core/resources/common/process_chain.py]: See other question.

File: src/actinia_core/resources/common/process_chain.py (hunk 1 above)
Comment (mmacata) [reply src/actinia_core/resources/common/process_chain.py]: Very good hint, there was some process-chain confusion about that value. We found it thanks to your hint and fixed it. I changed above to a boolean

**Comment by mmacata** [comment]
Ok thank you, I'll merge now

**Pull request merged**